helix-query.yaml

```diff
@@ -0,0 +1,12 @@
+version: 1
+
+indices:
+  articles:
+    include:
+      - '/articles/**'
+    target: /query-index.xlsx
+    properties:
```

Comment:
We should probably add image, description and maybe keywords to be on the safe side

Reply:
added extra properties